### PR TITLE
init command: populate default charmcraft.yaml (CRAFT-330)

### PR DESCRIPTION
**Pull request by cjp256**
Initialize charmcraft.yaml with type "charm" and an Ubuntu 20.04
bases configuration.

Signed-off-by: Chris Patterson <chris.patterson@canonical.com>

### CODE REVIEW

**Comment by cjp256** [comment]
Requires #414

File: charmcraft/templates/init/charmcraft.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Learn more about config at: https://juju.is/docs/sdk/config
```
Comment (jnsgruk) [inline charmcraft/templates/init/charmcraft.yaml.j2:1]: This link should probably be: https://juju.is/docs/sdk/charmcraft-config

Comment (cjp256) [reply charmcraft/templates/init/charmcraft.yaml.j2]: Yes indeed, thanks! Done!

**Review by sergiusens — APPROVED** [review]

**Review by facundobatista — APPROVED** [review]
Like it!

**Pull request merged**